README.md

```diff
@@ -11,12 +11,43 @@
 
 As programmers, we wonder about the world, come up with questions and then
 ask a computer to help us manipulate data to find answers. Many of those
-questions can be resolved by "polling" every member in a collection. We'd
-typically "poll" each number in the collection and feed that value to a
-calculation (French Word For _x_) _or_ aggregate that value into a running
-value (like a total).
+questions can be resolved by either:
 
-Here are some real-world questions:
+1. "Polling" every member in a collection and collecting the results into a
+   new collection through a process called "mapping" (or)
+2. "Polling" every member in a collection and joining those poll's results together
+   into a single value through a process called "reducing"
+
+Despite these fancy names of "map" and "reduce" we "map" and "reduce" all the
+time.
+
+For example, we might ask a collection of staff
+members whether they want "Sushi Menu 1," "Sushi Menu 2," or "vegan option"
+for the upcoming team meeting. Think back to our first paragraph: "polling
+every member in a collection and collecting the results into a new collection."
+That's "mapping," but you might not have called it by that name before.
+
+Or, what if we asked everyone how much cash they have on hand so that we
+can tip the delivery person for our team lunch (forgot to plan on that one!).
+We'll "poll each member in a collection and join those results (by addition)
+together" to find out how much cash we have between us. That's
+"reducing," but you might not have called it by that name before!
+
+### Uses of "map" and "reduce"
+
+Oftentimes we "map" or "reduce" a collection so that we can ask a question
+of that new collection.
+
+* Did more than half of the staff choose the vegan option?
+* Is the collected cash on hand greater or equal to 20% of the bill for staff lunch?
```

Comment:
I think this example is great, though I do wonder if tip is a universally applicable concept or if this will feel like a particularly American thing?

Reply:
I'm gonna roll the dice on this one. Pourboir and service fees are common throughout the world. We'll see if we can get specific resistance.